pkg/apis/imagepolicy/types.go

```diff
@@ -56,12 +56,18 @@ type ImageReviewContainerSpec struct {
 	// In future, we may add command line overrides, exec health check command lines, and so on.
 }
 
-// ImageReviewStatus is the result of the token authentication request.
+// ImageReviewStatus is the result of the review for the pod creation request.
 type ImageReviewStatus struct {
 	// Allowed indicates that all images were allowed to be run.
 	Allowed bool
 	// Reason should be empty unless Allowed is false in which case it
 	// may contain a short description of what is wrong.  Kubernetes
 	// may truncate excessively long errors when displaying to the user.
 	Reason string
+	// Annotations that should be added to the annotations in the attributes
+	// object of the admission controller request. The keys should be
+	// prefix-less (i.e., the admission controller will add an appropriate
+	// prefix).  Annotations are only added to the attributes if the the
+	// creation is allowed.
+	AttributeAnnotations map[string]string
```

Comment:
let's match the admission webhook response field name of `auditAnnotations`

Reply:
k, will do.